check-osp/conf/services_to_check.yaml.template

```diff
@@ -0,0 +1,25 @@
+---
+host1.example.com:
```

Comment:
How do you want to handle the different hosts for different instances of OSP? Different files for each instance? Or should we have a field which also specifies the OSP hostname? 
```yaml
osp-hostname:
  host1:
    services...
  host2:
    services...
osp-hostname2:
  host1:
    services..
```

Reply:
I was thinking different files, we can have files named after provider with all of its hosts and services. What say? Good separation or not?

Reply:
hmm, the only problem I can think of with that is that we will have to define a separate command for each provider in the pack, which we don't want to do. I think it'd be better to have the single file so that we will only have the 1 command.

Reply:
Got it. Let me do the update then.

Reply:
Upon further discussion offline, we decided to defer this.